src/rgw/rgw_auth_s3.cc

```diff
@@ -1022,6 +1022,32 @@ size_t AWSv4ComplMulti::recv_body(char* const buf, const size_t buf_max)
   return buf_pos;
 }
 
+size_t AWSv4ComplMulti::recv_body(char* const buf, const size_t buf_max)
+{
+  /* get total buf_max data from the stream, buf_max's default is  rgw_max_chunk_size
+   * we're getting from a client. */
+  size_t buf_pos = 0; 
+  size_t buf_incomplement_size = buf_max; //bul_max is 4M, exclude the last muiltpart
+
+  size_t len  = 0;
+  while (buf_incomplement_size > 0){
+          const auto read_len = recv_chunk_body( buf+buf_pos, buf_max);
```

Comment:
nit: please keep indentation at 2 spaces

Reply:
yes,  i will change the  current indentation  to two spaces . Thank you

Reply:
yes,  i will change the  current indentation  to two spaces . Thank you